### DISCUSSION TITLE
Feedback on DAO Index V0.9

### DISCUSSION THREAD

**Discussion by linxule** [Ledgerback/DGSF#3] [Ideas]
I have added DAO_Index_V0.9.1_XL under the Questionnaire folder. Let me know what you think of my comments and edits.

@kylefox1 @vincenzospaghetti

**Comment by kylefox1** [comment]
Thank you for adding it. I will check it out soon.

**Reply by kylefox1** [reply]
I read some of the comments. 

Did you get a chance to read through some of the assessments?

It should help with thinking about whether to add, remove, or modify a question.

**Reply by kylefox1** [reply]
Also, for any new questions you want to add, include them in the Question_Pool-DAO_Index-2022_0902-V0.1 table.

**Reply by linxule** [reply]
I will check against the assessment file. It might be helpful to has out these ideas first. This might be a difference in our workflows – I find it difficult to discuss via table formats. I think if it's just minor changes within questions, a table of questions might be helpful. But I think we are not there yet.

**Reply by linxule** [reply]
> I read some of the comments. 
> 
> 
> 
> Did you get a chance to read through some of the assessments?
> 
> 
> 
> It should help with thinking about whether to add, remove, or modify a question.

Yes. I did. I think mostly my suggestions are for responses to be mapped to slightly questions or broken down into multiple questions. Give it a read and let me know what you think.

**Reply by kylefox1** [reply]
> Yes. I did. I think mostly my suggestions are for responses to be mapped to slightly questions or broken down into multiple questions. Give it a read and let me know what you think.

I do not understand what you are saying here.

**Comment by linxule** [comment]
Hi @kylefox1 @vincenzospaghetti,

I've updated my draft for the DAO index and created an Airtable base([DAO Index_Draft_XL](https://airtable.com/appxwnyzem8EAbGtk/tblHVUkQTReLlfZm2/viwNIMm7TFEQqelHu?blocks=hide)) for you to review. The main objective is to map what @kylefox1 has created to different dimensions and structures of **organizational design**. I believe it's valuable to keep both frameworks, and we can run analyses such as CFA to see how they compare. **These two frameworks offer different perspectives: one is more design-focused, while the other is more principle-focused. I'd love to hear your thoughts on how they compare.**

I've added some questions to the Airtable base. You'll notice that some original questions now have duplicates, as I've modified/expanded some questions, which you can see in the "**Questions_XL**" column. For a quicker view of the framework, you can check out the **"Organizational Design Structure" tab**. I've tried to minimize the number of questions, but I'm not sure if we can further reduce the number of organizational design dimensions at the moment. I would appreciate your help in suggesting what can be combined, especially regarding shared goals, stakeholder participation, algorithmic management, and human management.

**One key aspect missing from our questions is the "Tasks" dimension**, which covers task definition, allocation, and execution. Although these aspects are somewhat covered by delegated decision rights and their associated structures, I'd love to hear your thoughts on whether we should ask some questions related to these structures within the task dimension of organizational design.

I've added you to the Airtable base as creators. Feel free to create new views according to your preference. Please reach out to me if you encounter any issues accessing it.

Looking forward to your insights and suggestions on refining the DAO index further, particularly in the areas of combining organizational design dimensions and incorporating the task dimension. You are welcome to leave your comments on each row or use the "Comment" column.

Cheers,
Xule

**Reply by kylefox1** [reply]
Add your questions to the Question_Pool-DAO_Index-2022_0902-V0.1 table.

**Reply by kylefox1** [reply]
And make sure to check the Glossary table.

**Reply by kylefox1** [reply]
I like the idea of mapping the principles and questions to the fundamental problems of organizing.

Though, you may want to use the structure mentioned in [Collaborative organizational forms: on communities, crowds, and new hybrids](https://link.springer.com/article/10.1186/s41469-018-0036-3) as a starting point (especially refer to the community ideal type).

Also, refer to Kelsie Nabben's piece on [DAO Design Patterns](https://platform.coop/blog/dao-design-patterns/).

Task-related questions are probably too specific to be included.

**Reply by kylefox1** [reply]
And you probably want to remove Delegated Decision Rights as a dimension because it is too specific on how a DAO should design their governance system. In other words, it excludes any DAOs that do not use a delegated voting system (i.e., un/elected representatives).

**Reply by linxule** [reply]
> I like the idea of mapping the principles and questions to the fundamental problems of organizing.
> 
> 
> 
> Though, you may want to use the structure mentioned in [Collaborative organizational forms: on communities, crowds, and new hybrids](https://link.springer.com/article/10.1186/s41469-018-0036-3) as a starting point (especially refer to the community ideal type).
> 
> 
> 
> Also, refer to Kelsie Nabben's piece on [DAO Design Patterns](https://platform.coop/blog/dao-design-patterns/).
> 
> 
> 
> Task-related questions are probably too specific to be included.

DAOs are different from the traditional forms of community (lack monetary incentives) and hierarchy (needs three transitive levels of "nodes"). These studies are useful and I cite them in my own work. But within the domain of organizational design, the emerging consensus is that existing frameworks are insufficient in comprehensively capturing the unique characteristics of DAOs.

**Reply by linxule** [reply]
> And you probably want to remove Delegated Decision Rights as a dimension because it is too specific on how a DAO should design their governance system. In other words, it excludes any DAOs that do not use a delegated voting system (i.e., un/elected representatives).

Every organization has some form of delegation of decision-making rights. E.g., managers, seniors, staffs in accounting firms. It is an essential part of organizational design.

**Reply by linxule** [reply]
Some relevant foundational & recent organizational design articles: 
- Puranam, Phanish. 2018. The Microstructure of Organizations. Vol. 1. Oxford University Press. https://doi.org/10.1093/oso/9780199672363.001.0001.
- Williamson, Oliver E. 1967. “Hierarchical Control and Optimum Firm Size.” Journal of Political Economy 75 (2): 123–38. https://doi.org/10.1086/259258.
- Lee, Eucman, Ekin Ilseven, and Phanish Puranam. 2023. “Scaling Nonhierarchically: A Theory of Conflict‐free Organizational Growth with Limited Hierarchical Growth.” Strategic Management Journal, July, smj.3541. https://doi.org/10.1002/smj.3541.
- Radner, Roy. 1993. “The Organization of Decentralized Information Processing.” Econometrica 61 (5): 1109–46. https://doi.org/10.2307/2951495.
- Dobrajska, Magdalena, Stephan Billinger, and Samina Karim. 2015. “Delegation Within Hierarchies: How Information Processing and Knowledge Characteristics Influence the Allocation of Formal and Real Decision Authority.” Organization Science 26 (3): 687–704. https://doi.org/10.1287/orsc.2014.0954.
- Burton, R., D. D. Håkonsson, J. Nickerson, P. Puranam, Maciej Workiewicz, and Todd R. Zenger. 2017. “GitHub: Exploring the Space between Boss-Less and Hierarchical Forms of Organizing.” Journal of Organization Design 6: 1–19. https://doi.org/10/gm2m52.
- Zhou, Yue Maggie. 2013. “Designing for Complexity: Using Divisions and Hierarchy to Manage Complex Tasks.” Organization Science 24 (2): 339–55. https://doi.org/10.1287/orsc.1120.0744.
- Hsieh, Ying-Ying, and Jp Vergne. 2022. “The Future of the Web? The Coordination and Early-Stage Growth of Decentralized Platforms.” Strategic Management Journal, August. https://doi.org/10.1002/smj.3455.
- Hsieh, Ying-Ying. 2018. “The Rise of Decentralized Autonomous Organizations: Coordination and Growth within Cryptocurrencies.” Doctoral dissertation, Canada: The University of Western Ontario.

**Reply by kylefox1** [reply]
> DAOs are different from the traditional forms of community (lack monetary incentives) and hierarchy (needs three transitive levels of "nodes"). These studies are useful and I cite them in my own work. But within the domain of organizational design, the emerging consensus is that existing frameworks are insufficient in comprehensively capturing the unique characteristics of DAOs.

So, are you saying that all DAOs must have monetary incentives? 

In other words, it is not possible for any DAO to exist that solely relies on non-monetary incentives?

What are these unique characteristics of DAOs (beyond just blockchain)?

> DAOs are different from the traditional forms of community (lack monetary incentives) and hierarchy (needs three transitive levels of "nodes"). These studies are useful and I cite them in my own work. But within the domain of organizational design, the emerging consensus is that existing frameworks are insufficient in comprehensively capturing the unique characteristics of DAOs.

So are you coming up with a new framework for capturing the unique characteristics of DAOs? Or is the framework you mentioned from an existing article?

**Reply by kylefox1** [reply]
> Every organization has some form of delegation of decision-making rights. E.g., managers, seniors, staffs in accounting firms. It is an essential part of organizational design.

I understand that. But your questions focused on delegate voting systems (governance specifically), which is too specific.

What you described in your sentence refers more to managerial roles (i.e., operational authority), rather than governance rights.
